.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,39 @@
+name: Node benchmarks
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: benchmarks
+      run: |
+        npm install
+        npm start y 100 10 40
+        npm start y 100 10 40
```

Comment:
Why are there run twice?

Reply:
Sorry that was my interpretation of README.md `two rounds; one to warm-up, one to measure` ;)

I gather that I should only run them once, and this is taken care of by `autocannon`?

Reply:
I have updated the `PR` to only run the benchmarks once.